packages/admin-ui/src/pages/support/components/PortsStatusTable.tsx

```diff
@@ -1,13 +1,12 @@
-import { useApi } from "api";
 import Button from "components/Button";
 import ErrorView from "components/ErrorView";
 import Loading from "components/Loading";
 import Ok from "components/Ok";
 import SubTitle from "components/SubTitle";
 import Switch from "components/Switch";
 import React, { useState } from "react";
+import { useApi } from "api";
 import { Table } from "react-bootstrap";
```

Comment:
You should never import react-bootstrap components from the root to ensure tree-shaking works well. See https://react-bootstrap.github.io/getting-started/introduction/#importing-components